pages/documentation/developer/tutorials/network_modifications_groovy.md

```diff
@@ -0,0 +1,95 @@
+# Tutorial on how to use network modifications methods in Groovy
+
+In this tutorial, you will find three scripts explaining how to add a load and a line with the associated topology to a network and how to remove a load.
+
+The network element is added to the network and the switches are also created to connect the element
+to a given busbar section with a closed disconnector and a breaker. Open switches are also added to every parallel busbar sections.
+When removing an element, the element itself is removed as well as all the breakers and disconnectors.
+
+Other similar functions are available to add every kind of injections and branches.
+
+NB: The design of these scripts is not groovy-friendly. The methods will be wrapped later.
+
+# CreateLoadBay.groovy
+
+In this script, a node/breaker network is first loaded. Then a load is added on the busbar section BBS1 of the voltage level named S1VL2 and the network is exported.
+
+Before adding the load, the voltage level S1VL2 is:
+
+![S1VL2](img/network_modifications_groovy/nb_network_s1vl2.png)
+
+The script is adding the load between the transformer TWT (order position 10) and the VSC converter station VSC1 (order position 20). For that, the order position is put to 15 when creating the modification.
+It is also possible to specify the direction of the load. Here it is not, so by default it is directed to the bottom.
```

Comment:
I would say: "It is also possible to specify the direction of the load. **By default,** it is directed to the bottom."